src/core/Password.cpp

```diff
@@ -1,76 +1,133 @@
 #include "Password.hpp"
-#include "hyprlock.hpp"
 #include "../helpers/Log.hpp"
+#include "hyprlock.hpp"
 
-#include <unistd.h>
+#include <security/_pam_types.h>
```

Comment:
Breaks build on FreeBSD which uses OpenPAM (also used on NetBSD, DragonFly, macOS):
```c++
src/core/Password.cpp:5:10: fatal error: 'security/_pam_types.h' file not found
    5 | #include <security/_pam_types.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
```

Try using `__has_include()` like a few lines below:
```suggestion
#if __has_include(<security/_pam_types.h>)
#include <security/_pam_types.h>
#endif
```